catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/operation/JdbcDatabaseOperations.java

```diff
@@ -77,6 +77,18 @@ public List<String> listDatabases() {
     }
   }
 
+  /**
+   * Load the schema with the given name.
```

Comment:
could you add some description about how shemaName maps to MySQL and PG?

Reply:
OK